src/comments_docsource.rs

```diff
@@ -231,12 +200,30 @@ impl CommentsDatabase {
             self.add_to_cache(*hash, defs);
         }
 
-        let out = bincode::serialize(self)?;
-        std::fs::write(&cache_path, out)?;
-
         Ok(true)
     }
 }
+impl Cache for CommentsDatabase {}
```

Comment:
This would ofc be prettier as a `#[derive(Cache)]` macro, although those are kind of annoying to implement. Generally something to keep in mind, but i don't think we need to necessarily do that now